api/office/models.py

```diff
@@ -6,10 +6,9 @@
 from api.location.models import Location  # noqa: F401
 
 
-class Block(Base, Utility):
-    __tablename__ = 'blocks'
+class Office(Base, Utility):
+    __tablename__ = 'offices'
     id = Column(Integer, primary_key=True)
     name = Column(String, nullable=False)
     location_id = Column(Integer, ForeignKey('locations.id'))
```

Comment:
Please include **location = relationship('Location')** for backward relationship for office with location.
Also rename the **block** package(folder) to **office**

Reply:
When that is done it create's some sort of loop.

Reply:
I'm not following because we had this when it was still block.

Reply:
@PatrickCmd The backward relationship between entities created an infinite sort of nested object.

Reply:
@SannyTee Yes it is the practice with graphql. So doing query for office and you need a location for each, backward relationship does it and if you inspect well, you will find this implemented in each model.

Reply:
@PatrickCmd from a user point of view, if the backward relationship is included, the graphql docs will reflect the change and it can be confusing for someone trying to consume the API.

Reply:
Thanks, @PatrickCmd. From the design specifications, I can see why the backward relationship is needed. The PR will be updated.